RNTester/RNTesterUnitTests/RCTConvert_UIColorTests.m

```diff
@@ -0,0 +1,200 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+#import <XCTest/XCTest.h>
+
+#import <React/RCTConvert.h>
+
+@interface RCTConvert_NSColorTests : XCTestCase
+
+@end
+
+@implementation RCTConvert_NSColorTests
+
+- (void)testColor
+{
+  id json = RCTJSONParse(@"{ \"semantic\": \"lightTextColor\" }", nil);
+  UIColor *value = [RCTConvert UIColor:json];
+  XCTAssertEqualObjects(value, [UIColor lightTextColor]);
+}
+
+- (void)testColorFailure
+{
+  id json = RCTJSONParse(@"{ \"semantic\": \"bogusColor\" }", nil);
+
+  __block NSString *errorMessage = nil;
+  RCTLogFunction defaultLogFunction = RCTGetLogFunction();
+  RCTSetLogFunction(^(__unused RCTLogLevel level, __unused RCTLogSource source, __unused NSString *fileName, __unused NSNumber *lineNumber, NSString *message) {
+    errorMessage = message;
+  });
+
+  UIColor *value = [RCTConvert UIColor:json];
+
+  RCTSetLogFunction(defaultLogFunction);
+
+  XCTAssertEqualObjects(value, nil);
+  XCTAssertTrue([errorMessage containsString:@"labelColor"]); // the RedBox message will contain a list of the valid color names.
+}
+
+- (void)testFallbackColor
+{
+  id json = RCTJSONParse(@"{ \"semantic\": \"unitTestFallbackColorIOS\" }", nil);
```

Comment:
why do we need the unit test specific semantic colors?

Reply:
Its a test of the fallback mechanism itself in RCTConvert.   'unitTestFallbackColorIOS' is a special debug only semantic color name (in react-native) that falls back to [UIColor blueColor].